src/palette.st.css

```diff
@@ -28,11 +28,13 @@
   success:	value(G10);
   warning:	value(Y10);
   danger:	value(R10);
+  error: value(R00);
   premium:	value(P10);
   urgent: value(O10);
   standard:  value(B20);
   general: value(D20);
   neutral: value(D40);
+  white: value(WHITE);
```

Comment:
white is not used anywhere, so please remove it

Reply:
my bad, done